var/spack/repos/builtin/packages/py-enum34/package.py

```diff
@@ -29,7 +29,6 @@ class PyEnum34(PythonPackage):
     """Python 3.4 Enum backported to 3.3, 3.2, 3.1, 2.7, 2.6, 2.5, and 2.4."""
```

Comment:
I'm not sure if that's what we want, because ```py-enum34``` should not be installed at all for ```python@3.4:```, it's more a "only install for" instead of a "if installed install also", or am I missing something?